products/unity.md

```diff
@@ -36,7 +37,7 @@ releases:
 
 -   releaseCycle: "2021"
     lts: true
-    releaseDate: 2021-03-22
+    releaseDate: 2022-04-11
```

Comment:
Wrong: https://unity.com/releases/editor/whats-new/2021.1.0

Link to https://github.com/endoflife-date/release-data/pull/149.